coriolis/providers/backup_writers.py

```diff
@@ -252,16 +252,21 @@ def _copy_helper_cmd(self, ssh):
             finally:
                 sftp.close()
 
-    @utils.retry_on_error()
+    @utils.retry_on_error(sleep_seconds=30)
```

Comment:
The arg is actually used in the method.
I see there's an `auth_timeout` arg to Client.connect(), but I presume it's for timeout during the initial connection and won't help us here.
http://docs.paramiko.org/en/2.6/api/client.html

Reply:
of no use. Auth timeout is for situations where client waits for the server to authenticate it, and takes longer than usual. If authentication fails due to bad credentials (ssh key not yet inserted into user's home), it fails instantly.